scripts/scripts.nut

```diff
@@ -31,255 +31,235 @@ function scripts_handle(name, ...) {
 
   // Add a dummy for the `this` pointer
   local a = vargv;
-  a.insert(0, null)
+  a.insert(0, null);
 
   foreach(script in scripts) {
 
-    local script_name = script[0]
-    local script_context = script[1]
+    local script_name = script[0];
+    local script_context = script[1];
 
     if (!(name in script_context)) {
-      //print("  - " + script_name)
+      //print("  - " + script_name);
       continue;
-    } else {
-      //print("  + " + script_name)
+    }
+    else {
+      //print("  + " + script_name);
     }
 
     local f = script_context[name];
 
     // Fixup `this` pointer
-    a[0] = script_context
+    a[0] = script_context;
 
     try {
       local result = f.acall(a);
 
       //FIXME: handle result conflicts
-      final_result = result
-
-    } catch(exception) {
+      final_result = result;
+    }
+    catch(exception) {
       print(exception);
       print(getstackinfos(1).locals);
       throw exception;
     }
-
-
   }
-  return final_result
+  return final_result;
 }
 
 class Hooks {
 
-function onServerStart() {
-  scripts_handle("onServerStart")
-}
+  function onServerStart() {
```

Comment:
`class` acts as a namespace. formatting of these hooks is also only changed once implemented.

I suggest to not edit this portion of the file.